packages/http-smoke-testing/src/Annotation/DataSet.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\HttpSmokeTesting\Annotation;
+
+use Doctrine\Common\Annotations\Annotation\Target;
+
+/**
+ * @Annotation
+ * @Target({"METHOD"})
+ */
+class DataSet
+{
+    /** @var int */
```

Comment:
```suggestion
    /** 
      * @var int
      */
```

Reply:
You've added to many spaces. But I will change it into multiline.